### PR TITLE
Adding test fixtures and three simple tests

### PR DESCRIPTION
**Pull request by darkwing**
This PR will be iterated on, but this gets two simple tests into the repo.

### CODE REVIEW

**Comment by akashnimare** [comment]
Try using '/html/index.html' instead of index.html

**Comment by openjck** [comment]
:tada: :balloon: :confetti_ball: :tada: :balloon: :confetti_ball: :tada: :balloon: :confetti_ball: :tada: :balloon: :confetti_ball: :tada: :balloon: :confetti_ball: :tada: :balloon: :confetti_ball:

**Comment by openjck** [comment]
:heart: config.js

File: package.json [hunk 1]
```diff
@@ -19,6 +19,10 @@
         "name": "Luke Crouch",
         "email": "luke.crouch@gmail.com",
         "url": "http://groovecoder.com"
+    }, {
+        "name": "David Walsh",
+        "email": "dwalsh@mozilla.com",
+        "url": "http://davidwalsh.name"
```
Comment (openjck) [inline package.json]: :heart:

**Comment by openjck** [comment]
I'll do a complete review on Friday, but I can already tell this is epic. So excited!

**Comment by darkwing** [comment]
Updated!

File: app.js [hunk 2]
```diff
@@ -1,14 +1,19 @@
 'use strict';
 
 var hook = require('./hook');
+var config = require('./config');
+var logger = require('./logger');
+var path = require('path');
```
Comment (openjck) [inline app.js]: Just a small thing. In other files, we've put contributed modules above custom modules separated by one line.

https://github.com/mdn/discord/blob/master/processor.js#L3-10

**Pull request merged**